docs/elements.md

```diff
@@ -529,13 +556,88 @@ Attribute | Type | Default | Description
 `allow-blank` | boolean | false |  Whether or not an empty input box is allowed. By default, an empty input box will not be graded (invalid format).
 `blank-value` | string | 0 (zero) | Expression to be used as an answer if the answer is left blank. Only applied if `allow-blank` is `true`. Must follow the same format as an expected user input (e.g., same variables, etc.).
 `size` | integer | 35 | Size of the input box.
 `show-help-text` | boolean | true | Show the question mark at the end of the input displaying required input parameters.
 
+##### Descriptions of ``enable-function-groups`` selections
+
+The next listings document what named function definitions are included
+by enabling these individual groups:
+
+* ``trig-base``: Support for definitions of basic (non-hyperbolic) trigonometric functions
+                      including $cos$, $sin$, $tan$, $sec$, $cot$, $csc$,
+                      ``acos``, ``asin``, ``atan``,
+                      ``asec``, ``acot``, ``acsc``,
+                      ``arccos``, ``arcsin``,
+                      ``arctan``, and ``atan2``.
+* ``trig-hyperbolic``: Support for definitions of hyperbolic trig functions and their inverses
+                      including $cosh$, $sinh$, $tanh$, ``sech``, $coth$, ``csch``,
+                      ``acosh``, ``asinh``, ``atanh``,
+                      ``asech``, ``acoth``, and ``acsch``.
+* ``complex-variables``: Support for definitions of standard functions used
+                       with complex variables including ``arg``, ``mod``,
+                       ``re``, ``im``, and ``conj``.
+* ``logarithmic``: Support for definitions of common variants on notation for a few
+                      logarithm type functions including $log$, $ln$, ``lg``, ``log2``,
+                      ``log10``, $exp$.
+* ``prog-extended``: Support for definitions of functions that are otherwise common in
+                      discrete math and / or theory classes including
+                      $max$, $min$, ``ceil``, ``floor``,
+                      ``frac``, ``abs``, ``sgn``,
+                      $gcd$, ``lcm``, and ``sqrt``.
+* ``calculus-extended``: Support for definitions of function names and implementations
+                      that are common in problem sets for typical introductory courses in differential and integral calculus
+                      including $cos$, $sin$, $tan$, $sec$, $cot$, $csc$,
+                      ``acos``, ``asin``, ``atan``,
+                      ``asec``, ``acot``, ``acsc``,
+                      ``arccos``, ``arcsin``,
+                      ``arctan``, ``atan2``, $cosh$,
+                      $sinh$, $tanh$, ``sech``, $coth$, ``csch``,
+                      ``acosh``, ``asinh``, ``atanh``,
+                      ``asech``, ``acoth``, ``acsch``,
+                      $ln$, and $exp$.
```

Comment:
Yes, probably. The reason I started in on this PR was to add support for common function names you need to ask students questions in an intro integral calculus course. It happens that there is an alias available for this, e.g., more than one way to get the calc base support. Since I believe this is a common request (there were also some physicists asking for related functionality), we should cater to calc instructors and make their lives easy by enabling it all with just ``calculus-extended``. Makes good sense and me happy. 

Did I understand the question correctly?

Reply:
I still agree with my point about aliases for. subsets of the custom functions being a good thing, TM. As long as the alias does not duplicate the functionality of another verbatim. That is, if ``calculus-extended`` and ``physics-extended`` (say, if there were one) add the same set of functions, then it is nonsense. On the other hand, providing an interface that is easy to use for common (would be common) applications like calc or programming courses, or discrete math (and others that can be added later) is a best practice type argument.